xrviz/__init__.py

```diff
@@ -3,3 +3,11 @@
 from ._version import get_versions
 __version__ = get_versions()['version']
 del get_versions
+
+
+def example():
```

Comment:
I suggest an optional parameter `show=True`, which, when False, causes `dash` to be returned rather than calling `.show()` on it. Then, you can include a test for this function, and of course docstrings.

Reply:
I have added these changes. Thanks!